src/main/java/com/example/teampandanback/service/ProjectService.java

```diff
@@ -112,4 +115,51 @@ public ProjectCrewResponseDto readCrewList(Long projectId){
                 .build();
     }
 
+    @Transactional
+    public ProjectInvitedResponseDto invitedProject(ProjectInvitedRequestDto projectInvitedRequestDto, SessionUser sessionUser) {
+
+        String decodedString = null;
+        Long decodedLong = null;
+        try {
+            decodedString = aesEncryptor.decrypt(projectInvitedRequestDto.getInviteCode());
+            decodedLong = Long.parseLong(decodedString);
+        } catch (NumberFormatException e) {
+            throw new ApiRequestException("복호화 된 프로젝트ID가 숫자가 아닙니다. 프로젝트ID = " +decodedString);
+        } catch (Exception e) {
+            throw new ApiRequestException(e.getMessage());
```

Comment:
e.getMessage()를 프론트단에 그대로 보내주는 것은 문제가 있다고 생각합니다. 차라리 해당 내용은 로그로 출력했으면 좋겠다는 생각합니다.
```suggestion
            log.info(e.getMessage());
            throw new ApiRequestException("유효하지 않은 초대 코드 입니다.");
```

Reply:
동의합니다